docs/content/topics/scripting/brigadier.md

```diff
@@ -1,6 +1,5 @@
 ---
-linkTitle: The Brigadier Library
-title: The Brigade scripting API
+title: The Brigadier Library
```

Comment:
Didn't go too overboard reining in use of "API," but imho, "API," while not inaccurate, has connotations to it that don't hold for Brigadier, so I made the choice to s/API/Library/ just for the title here.